src/cascade/executor/priors_from_draws.py

```diff
@@ -0,0 +1,111 @@
+import itertools as it
+import numpy as np
+
+from cascade.core.log import getLoggers
+
+CODELOG, MATHLOG = getLoggers(__name__)
+
+
+def set_priors_from_draws(model, draws):
+    """Sets priors from posteriors of the *same model*.
+    Use this when you have fit fixed and then want to fit again for both.
+
+    Args:
+        model (Model): A complete model for this location. It will be modified.
```

Comment:
I can't think when we would keep the old one. This may not have a decent copy() command on the model, either. That's work that could be done on the model. I'm hoping to make a read_model command to read from the db. That would be a good time.

Reply:
Fair enough.